### ISSUE TITLE
Try out 'refined github' extension

### ISSUE THREAD

**Issue by iteles** [iteles/personal-goals#4]
Sometimes I will not read a notification on an important issue for several hours or even 2 or 3 days because I know I don't have time to deal with anything on it right now and want it to remain 'unread' so that I am reminded of it when I next come to my notifications.

It looks like [Refined Github](https://github.com/sindresorhus/refined-github) is open source, has tests, recently updated and will do this! Thanks to jbarget for the tip 🎉 

Aim: Use for 2 weeks and see if I use it/ for what  and whether it works as advertised.

**Comment by nelsonic** [comment]
I'm _very_ keen to read/see your findings on this! 😉

**Comment by iteles** [comment]
This doesn't actually work as it it supposed to. A notification only stays 'unread' for a short period of time.

However, it has a handy feature which show you who has 'reacted' to certain comments.

It also has a number of other features which I have yet to try out so will keep this for a little longer.

**Comment by iteles** [comment]
_Most_ of what I was using this for has now been implemented by github.
The only thing that hasn't is the visual representation of who has reacted to an issue:
![image](https://user-images.githubusercontent.com/4185328/48984172-5a252080-f0f0-11e8-9084-051558d17ed0.png)

I still find this quite time saving as when I uninstalled the extension for a short period, I found myself having to move the mouse around a lot as I still wanted to know who had reacted to the comments. Therefore, for now, still worth it so that I don't have to go hover over everything.
